src/Bounds.cpp

```diff
@@ -2476,10 +2474,8 @@ class BoxesTouched : public IRGraphVisitor {
             for (const auto &pair : cases) {
                 Expr c = pair.first;
                 Stmt body = pair.second;
-                const Call *call = c.as<Call>();
-                if (call && (call->is_intrinsic(Call::likely) ||
-                             call->is_intrinsic(Call::likely_if_innermost) ||
-                             call->is_intrinsic(Call::strict_float))) {
+                const Call *call = Call::as_tag(c);
```

Comment:
```if (const Call *call == Call::as_tag(c)) {```

Reply:
`call` is used below :) I made this change at first and had to revert it for that.